docs/developers/REVIEWS.md

```diff
@@ -22,14 +22,9 @@ Please fix any issue raised.
 It might take some time until your PR gets merged, depending on the release cycle and the type of
 your pull-request (bugfix, enhancement of existing code, new plugin, etc). Remember, it might be necessary to rebase your code before merge to resolve conflicts.
 
-Please read the review comments carefully, fix the related part of the code and/or respond in case there is anything unclear. If there is no activity in a pull-request or the contributor does not respond, we apply the following scheme:
+Please read the review comments carefully, fix the related part of the code and/or respond in case there is anything unclear. If there is no activity in a pull-request or the contributor does not respond, we are setting the `waiting for response` tag. __If there is no further activity our bot will automatically close the PR after two weeks!__ So in case you expect a longer period of inactivity or you want to abandon a pull-request, please let us know.
```

Comment:
```suggestion
Please read the review comments carefully, fix the related part of the code and/or respond in case there is anything unclear. Maintainers will add the `waiting for response` tag to PRs to make it clear we are waiting on the submitter for updates. __Once the tag is added, if there is no activity on a pull request or the contributor does not respond, our bot will automatically close the PR after two weeks!__ If you expect a longer period of inactivity or you want to abandon a pull request, please let us know.
```